shared/editor/embeds/GoogleForms.test.ts

```diff
@@ -0,0 +1,55 @@
+import GoogleForms from "./GoogleForms";
+
+describe("GoogleForms", () => {
+  const match = GoogleForms.ENABLED[0];
+  const match2 = GoogleForms.ENABLED[1];
+
+  test("to be enabled on long-form share links", () => {
+    expect(
+      "https://docs.google.com/forms/d/e/1FAIpQLSetbCGiE8DhfVQZMtLE_CU2MwpSsrkXi690hkEDREOvMu8VYQ/viewform?usp=sf_link".match(
+        match
+      )
+    ).toBeTruthy();
+    expect(
+      "https://docs.google.com/forms/d/e/1FAIpQLSetbCGiE8DhfVQZMtLE_CU2MwpSsrkXi690hkEDREOvMu8VYQ/viewform".match(
+        match
+      )
+    ).toBeTruthy();
+    expect(
+      "https://docs.google.com/forms/d/e/1FAIpQLSetbCGiE8DhfVQZMtLE_CU2MwpSsrkXi690hkEDREOvMu8VYQ/viewform?embedded=true".match(
+        match
+      )
+    ).toBeTruthy();
+  });
+
+  test("to be enabled on edit links", () => {
+    expect(
+      "https://docs.google.com/forms/d/1zG75dmHQGpomQlWB6VtRhWajNer7mKMjtApM_aRAJV8/edit".match(
+        match
+      )
+    ).toBeTruthy();
+  });
+
+  test("to be enabled on short-form share links", () => {
```

Comment:
Note: Unlike long form share links, short form links don't have `?embedded=true` variant. Should we support these?

Reply:
It would be nice, I suppose to do so correctly we'd need to load the short link with a fetch request inside the embed component and get the 302 Location. This could then be used with the `embedded` variant

Reply:
> we'd need to load the short link with a fetch request inside the embed component

Not sure if that would violate same origin policy, but I can give it a shot.

Reply:
Obviously, failed due to cross origin request being blocked. Should we do it via server? Let me know in case you've any other _safer_ hacks up your sleeve :smile:

Reply:
Probably don't want dedicated endpoints for this, lets just remove the short url matching to get this first version landable :)

Reply:
Cool! It's more fun to remove code, especially when that dumps a _not so important_ feature. I totally second this stand :smile: